build.sbt

```diff
@@ -88,7 +88,16 @@ lazy val utilDependencies = Seq(
   "org.slf4j" % "slf4j-api" % "1.7.25",
   "org.apache.commons" % "commons-compress" % "1.15",
   "org.tensorflow" % "tensorflow" % "1.8.0",
-  "com.amazonaws" % "aws-java-sdk-s3" % "1.11.313"
+  // Enable the following for tensorflow support
+  //"org.tensorflow" % "libtensorflow" % "1.8.0",
+  //"org.tensorflow" % "libtensorflow_jni_gpu" % "1.8.0",
+  "com.amazonaws" % "aws-java-sdk-s3" % "1.11.313" exclude("org.slf4j", "slf4j-log4j12") exclude("log4j", "log4j"),
+  "org.apache.pdfbox" % "pdfbox" % "2.0.9" exclude("org.slf4j", "slf4j-log4j12") exclude("log4j", "log4j"),
+  "net.sourceforge.tess4j" % "tess4j" % "4.0.2" exclude("org.slf4j", "slf4j-log4j12") exclude("log4j", "log4j"),
```

Comment:
All these excludes can be replaced with only one exclude on Tesseract dependency, like this,
"net.sourceforge.tess4j" % "tess4j" % "4.0.2" exclude("org.slf4j", "slf4j-log4j12-1.7.")

Reply:
I don't remember making this change, but okay